cmd/vclusterctl/cmd/create.go

```diff
@@ -221,6 +226,27 @@ func (cmd *CreateCmd) Run(ctx context.Context, args []string) error {
 		if len(cmd.Values) <= 0 && migratedValues != "" {
 			return fmt.Errorf("abort upgrade: no values files were provided while the current virtual cluster is running with an old values format (< v0.20)\nConsider updating your config values to the following new v0.20 format:\n\n%s", migratedValues)
 		}
+
+		// (ThomasK33): Since no value was migrated, it's safe to assume that the previous installation was a 0.20+ one
+		if release != nil {
+			releaseValues, err := yaml.Marshal(release.Config)
+			if err != nil {
+				return err
+			}
+
+			cfg := config.Config{}
+			if err := cfg.DecodeYAML(bytes.NewReader(releaseValues)); err != nil {
+				return err
+			}
+
+			if ok, err := setup.CheckUsingHelm(ctx, cmd.kubeClient, args[0], cmd.Namespace, cmd.Distro, cfg.BackingStoreType()); err != nil {
```

Comment:
Since people can configure the distro via values now, I think this can be problematic here. I would leave out the CLI check for now